doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -396,6 +396,7 @@ Matter samples
 * :ref:`matter_smoke_co_alarm_sample` sample:
 
   * Added support for ICD dynamic SIT LIT switching (DSLS).
+  * Added support for system off configuration for the :ref:`zephyr:nrf54l15dk_nrf54l15` target.
```

Comment:
Add the changelog entry as a comment to my https://github.com/nrfconnect/sdk-nrf/pull/19240 and remove it from this PR.